src/Altinn.App.Core/Implementation/ProcessEngine.cs

```diff
@@ -34,7 +34,7 @@ public class ProcessEngine : IProcessEngine
         /// </summary>
         public async Task<ProcessChangeContext> Next(ProcessChangeContext processChange)
         {
-            string currentElementId = processChange.Instance.Process.CurrentTask?.ElementId;
+            string? currentElementId = processChange.Instance.Process.CurrentTask?.ElementId;
```

Comment:
Fixed while working on gateways in the process. Revert should resolve the merge conflicts
```suggestion
            string currentElementId = processChange.Instance.Process.CurrentTask?.ElementId;
```